code/src/Components/SurveyRadio.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+const SurveyRadio = (props) => {
+    const { type, setType } = props;
+
+    return (
+        <div className="radio-form" >
+            <p className="radio-form-headline">How would you prefer to live?</p>
+            <label>
+                <input className="radio-input"
+                id="radio"
```

Comment:
If you want to associate the `label` element with the input element you should set a `for` attribute to the `label ` that has the same value as the `input id`🙋‍♀️  This link helped me https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label